Dockerfile

```diff
@@ -1 +1,79 @@
 FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base_java8:latest
+LABEL ORG="Armedia LLC" \
```

Comment:
@naveen-armedia I think we said we would not maintain e.g. `ark_base_java8` but instead add Java right into the images that need it?  I still prefer a more layered set of images, but I believe the team ended up disagreeing with me.  So we would add Java 8 as a layer in this image instead of starting from `ark_base_java8`.  Or would that be another MR?